src/main/java/org/jabref/gui/preferences/external/ExternalTabViewModel.java

```diff
@@ -39,7 +40,7 @@ public class ExternalTabViewModel implements PreferenceTabViewModel {
     private final BooleanProperty autoOpenAttachedFoldersProperty = new SimpleBooleanProperty();
     private final ListProperty<PushToApplication> pushToApplicationsListProperty = new SimpleListProperty<>();
     private final ObjectProperty<PushToApplication> selectedPushToApplicationProperty = new SimpleObjectProperty<>();
-    private final StringProperty citeCommandProperty = new SimpleStringProperty("");
+    private final StringProperty citeCommandProperty = new SimpleStringProperty();
```

Comment:
Better use empty string to avoid possible npe

Reply:
Please tell this the Siedlerchr of 3 hours ago :p